examples/decoding/plot_decoding_xdawn_transformer.py

```diff
@@ -40,9 +40,10 @@
                 add_eeg_ref=False, verbose=False)
 
 X = epochs.get_data()
-y = label_binarize(epochs.events[:, 2], classes=[1, 3])
+y = label_binarize(epochs.events[:, 2], classes=[1, 3]).ravel()
 
 clf = make_pipeline(XdawnTransformer(n_components=3),
+                    Vectorizer(),
```

Comment:
travis is correct

https://github.com/kingjr/mne-python/blob/51be050157af289225b78ed967cd957a14493ad8/examples/decoding/plot_decoding_xdawn_transformer.py